### PR TITLE
added additonal notes to V3

### PR DESCRIPTION
**Pull request by lparadkar-rocket**
Describe your pull request here: Added some more notes for V3

List the file(s) included in this PR:
v3_0_0.md
After creating the PR, follow the instructions in the comments.

### CODE REVIEW

**Review by 1000TurquoisePogs — CHANGES_REQUESTED** [review]

File: docs/whats-new/release-notes/v3_0_0.md [hunk 1]
```diff
@@ -23,12 +23,47 @@ A series of Zowe V3 LTS office hours are available which detail all changes intr
...
+* RBAC users should update `allowedPlugins.json` for the new plugin ID of the Zowe v3. ([#3984](https://github.com/zowe/zowe-install-packaging/pull/3984)) 
+
+#### New features and enhancements in Zowe Application Framework
+#### Zlux App Server
+* Updated the Desktop Angular version from 12 to 18.  Angular apps written for V2 will need to be upgraded to work in V3. `Iframe` and `React` is unaffected. ([#639](https://github.com/zowe/zlux-app-manager/pull/639))
+* The V2 Desktop is included alongside the V3 Desktop as a temporary compatibility option for the duration of the V2 maintenance schedule. Any user can switch to the V2 Desktop via the `?use-v2-desktop=true` parameter. ([#639](https://github.com/zowe/zlux-app-manager/pull/639))
+
+#### Zlux App Server
```
[the remark this answers is not in the document]
Comment (lparadkar-rocket) [reply docs/whats-new/release-notes/v3_0_0.md:42]: Thanks, I made the change.

File: docs/whats-new/release-notes/v3_0_0.md [hunk 2]
```diff
@@ -23,12 +23,47 @@ A series of Zowe V3 LTS office hours are available which detail all changes intr
...
 
 * Introducing _Zowe Server Install Wizard_, a wizard to simplify installation and configuration of Zowe server side components via a UI.
+* Added explanation to possible error message when checking z/OSMF setting. ([#3956](https://github.com/zowe/zowe-install-packaging/pull/3956))
 
 ### Zowe Application Framework
 
 #### Breaking changes in Zowe Application Framework
 
 * Updated Angular to Version 18 from Version 12. Apps built upon Angular, excluding iframe apps, will need to be updated to be compatible with the V3 Desktop.
+* The Zowe YAML now requires the `zaas` component for the API Mediation Layer to work. See `example-zowe.yaml` for new component values.
```
[the remark this answers is not in the document]
Comment (lparadkar-rocket) [reply docs/whats-new/release-notes/v3_0_0.md]: Updated as per your comment.

**Review by janan07 — APPROVED** [review]

File: docs/whats-new/release-notes/v3_0_0.md [hunk 3]
```diff
@@ -23,12 +23,46 @@ A series of Zowe V3 LTS office hours are available which detail all changes intr
...
 
 * Introducing _Zowe Server Install Wizard_, a wizard to simplify installation and configuration of Zowe server side components via a UI.
+* Added explanation to possible error message when checking z/OSMF setting. ([#3956](https://github.com/zowe/zowe-install-packaging/pull/3956))
 
 ### Zowe Application Framework
 
 #### Breaking changes in Zowe Application Framework
 
 * Updated Angular to Version 18 from Version 12. Apps built upon Angular, excluding iframe apps, will need to be updated to be compatible with the V3 Desktop.
+* The Zowe YAML attribute `components.gateway.apiml.security.auth.zosmf.serviceId` for the `gateway` component has changed from `zosmf` to `ibmzosmf`. This may impact Zowe Clients. See `example-zowe.yaml` for new component values. For more information on this change, please see the API Mediation Layer's 3.0.0 Breaking Changes.
```
Comment (1000TurquoisePogs) [inline docs/whats-new/release-notes/v3_0_0.md:33]: There is still APIML content here that does not belong here

File: docs/whats-new/release-notes/v3_0_0.md [hunk 4]
```diff
@@ -23,12 +23,46 @@ A series of Zowe V3 LTS office hours are available which detail all changes intr
...
+* The Zowe YAML parameter `components.gateway.apiml.security.auth.zosmf. jwtAutoconfiguration` for the `gateway` component has changed. The value `auto` is no longer allowed, and you must choose either the default of `jwt` or `ltpa` depending on if your z/OSMF is set up for JWT use as recommended See `example-zowe.yaml` for new component values. For more information on this change, please see the API Mediation Layer's 3.0.0 Breaking Changes.
```
Comment (1000TurquoisePogs) [inline docs/whats-new/release-notes/v3_0_0.md:34]: There is still APIML content here that does not belong here

**Review by 1000TurquoisePogs — CHANGES_REQUESTED** [review]

File: docs/whats-new/release-notes/v3_0_0.md (hunk 3 above)
Comment (lparadkar-rocket) [reply docs/whats-new/release-notes/v3_0_0.md]: Have updated  the content as per your comment. Please review and approve.

File: docs/whats-new/release-notes/v3_0_0.md (hunk 4 above)
Comment (lparadkar-rocket) [reply docs/whats-new/release-notes/v3_0_0.md]: Has been removed from this section. Please review and approve.

**Review by 1000TurquoisePogs — APPROVED** [review]

**Pull request merged**